distributions/application/src/main/java/com/connexta/replication/application/Main.java

```diff
@@ -30,7 +32,12 @@
 @EnableConfigurationProperties
 public class Main {
 
+  private static final Logger LOGGER = LoggerFactory.getLogger(Main.class);
+
+  @SuppressWarnings("squid:S2189")
   public static void main(String[] args) throws Exception {
+    // Need to load some system properties before anything else is done
+    loadBootSystemProperties();
```

Comment:
We were having problems putting the ssl stuff in the application.yml because it wasn't loading the properties in time for some of the classes. Would love to have another way to do this.